### ISSUE TITLE
Cannot send singular title/subtitle/times with Adventure

### ISSUE THREAD

**Issue by kezz** [Minestom/Minestom#191]
### What is the current behavior?
As `Title` does not accept null components, there is no way to set individual elements of a title without resending the whole set of title packets.

### Additional information / Possible thoughts on why this bug is happening
This is not intended behaviour - the vanilla client does not have the possibility to set individual parts of a title without setting the entire element. Additionally, in 1.17 this will be hard coded. It's perhaps best to leave this as it is and mark with `wontfix` to prep for 1.16 and make the legacy chat methods of sending titles not delegate to the Adventure method.

**Comment by kezz** [comment]
Waiting on https://github.com/KyoriPowered/adventure/issues/314.

**Comment by TheMode** [comment]
> This is not intended behaviour - the vanilla client does not have the possibility to set individual parts of a title without setting the entire element. Additionally, in 1.17 this will be hard coded. It's perhaps best to leave this as it is and mark with wontfix to prep for 1.16 and make the legacy chat methods of sending titles not delegate to the Adventure method.

After checking the snapshot code, it appears that the only protocol difference is that each title component (title/subtitle/actionbar/time) will get their own packets instead of sharing a single code. Meaning that the behavior is unlikely to change
